openerp/addons/base/res/res_users.py

```diff
@@ -748,6 +750,9 @@ def update_user_groups_view(self, cr, uid, context=None):
                     app_name = app and app.name or _('Other')
                     xml2.append(E.separator(string=app_name, colspan="4", **attrs))
                     for g in gs:
+                        technical_group_id = self.pool['ir.model.data'].xmlid_to_res_id(cr, SUPERUSER_ID, 'base.group_no_one')
```

Comment:
Do the xmlid_tores_id only once + do a check once: is_technical = technical in groups .